services/events/project/scripts/event_tool.py

```diff
@@ -0,0 +1,21 @@
+from project import db
+from project.api.models import Event
+from datetime import datetime
+
+
+def soft_delete_events(source):
+    now = datetime.now().isoformat()
```

Comment:
Doesn't really make too much of a difference I can see right now but should this not be inside the loop so the timestamps are more accurate and not all identical when marking them 'deleted'?

Reply:
Good call; The soft deleted `deleted_at` was intended to be identical to show they were updated in the same batch but the `updated_at` field should contain the time it was actually updated. Will update